tutorcruncher/hermes
# sales round-robin with location

in `choose-sales-person`﻿ we now get the location from the header


- also added sells gb, eu, us, au, row to Admin model


### to test
- run make restore-from-live
- run uvicorn
- run aerich upgrade
- navigate to localhost:8000/login (after the restore you should be authorized)
- then navigate to /admin and edit an admin to have sells GB
- then navigate to http://localhost:8000/choose-roundrobin/sales/?plan=payg and get that admin
